man/journal-upload.conf.xml

```diff
@@ -63,20 +63,24 @@
       <varlistentry>
         <term><varname>Compression=</varname></term>
 
-        <listitem><para>Compression algorithm to be applied to logs data before sending.
-        Takes one of <literal>none</literal>, <literal>zstd</literal>, <literal>xz</literal>, and <literal>lz4</literal>.
-        Defaults to <literal>none</literal>.</para>
+        <listitem><para><literal>:</literal>-separated values of compression algorithm and level to be applied to logs data before sending.
+        Key one of <literal>none</literal>, <literal>zstd</literal>, <literal>xz</literal>, and <literal>lz4</literal>.
+        Defaults to <literal>none</literal>. Value is expected to be a positive integer.
+        Defaults to unset, and the default compression level of the specified algorithm will be used.</para>
+
+        <para>Even with compression enabled initial requests are sent without compression. It becomes effective either if
+        <literal>ForceCompression=</literal> is set to <literal>true</literal> or server response contains
+        <literal>Accept-Encoding</literal> headers with list of compression algorithms, which are configured and supported by
+        systemd-journal-upload.</para>
 
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><varname>CompressionLevel=</varname></term>
+        <term><varname>ForceCompression=</varname></term>
```

Comment:
sorry, but this is not OK. In systemd we want "perfect" PRs, where each commit makes sense on its own, and doesn't undo/fix any commit that is part of the very same PR. But here you first add Compressionlevel (incl. docs!), just to delete it again in the 2nd commit.

Please get this right n the first commit!